### PR TITLE
copy change, added grow open source image

### PR DESCRIPTION
**Pull request by cd10012**
##### Description

This aims to solve issue #238 just changing the body copy on the retail portion of the site. Let me know if you'd like to add more assets or put something on the homepage.
##### Checklist

- [x] changes don't break existing behavior
(https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
UI on the retail section of the site, changed body copy throughout the homepage and about section of the site. Updated the example.png asset since the badge has changed

##### Testing
I kept the structure of the HTML and just replaced assets and copy, I've viewed it at multiple screen widths and the responsive grids and styles remain attached.

### CODE REVIEW

**Comment by owocki** [comment]
a few comments
1. mind sizing down grow_open_source.png? its 1.2 MB (too large for web)
2. can you resolve the merge conflicts?
3. for some reason, i cant run `runserver` on this branch 

```
(gitcoin) kevinowocki@local /Users/kevinowocki/Sites/gitcoin/web/app~ % python3 manage.py runserver 0.0.0.0:8080
Performing system checks...

Unhandled exception in thread started by <function check_errors.<locals>.wrapper at 0x10f729840>
Traceback (most recent call last):
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/utils/autoreload.py", line 225, in wrapper
    fn(*args, **kwargs)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/core/management/commands/runserver.py", line 120, in inner_run
    self.check(display_num_errors=True)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/core/management/base.py", line 364, in check
    include_deployment_checks=include_deployment_checks,
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/core/management/base.py", line 351, in _run_checks
    return checks.run_checks(**kwargs)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/core/checks/registry.py", line 73, in run_checks
    new_errors = check(app_configs=app_configs)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/core/checks/urls.py", line 40, in check_url_namespaces_unique
    all_namespaces = _load_all_namespaces(resolver)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/core/checks/urls.py", line 57, in _load_all_namespaces
    url_patterns = getattr(resolver, 'url_patterns', [])
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/utils/functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/urls/resolvers.py", line 536, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/utils/functional.py", line 36, in __get__
    res = instance.__dict__[self.name] = self.func(instance)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/site-packages/django/urls/resolvers.py", line 529, in urlconf_module
    return import_module(self.urlconf_name)
  File "/Users/kevinowocki/code/.virtualenvs/gitcoin/lib/python3.6/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 978, in _gcd_import
  File "<frozen importlib._bootstrap>", line 961, in _find_and_load
  File "<frozen importlib._bootstrap>", line 950, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 655, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 678, in exec_module
  File "<frozen importlib._bootstrap>", line 205, in _call_with_frames_removed
  File "/Users/kevinowocki/Sites/gitcoin/web/app/app/urls.py", line 23, in <module>
    import dashboard.helpers
  File "/Users/kevinowocki/Sites/gitcoin/web/app/dashboard/helpers.py", line 70
    except ValidationError, e:
```

File: app/retail/templates/mission.html [hunk 1]
```diff
@@ -44,7 +44,7 @@
                     <br />
                     <span class="text-highlight-dark-blue">DISTRIBUTED MOVEMENT</span> AIMED TO <span class="text-highlight-dark-blue">DRIVE</span>
                     <br />
-                    <span class="text-highlight-dark-blue">CHANGE</span> AND <span class="text-highlight-pink">PUSH OPEN SOURCE</span> FORWARD
```
Comment (owocki) [inline app/retail/templates/mission.html]: itd be great to see the hero image of this page updated

**Comment by owocki** [comment]
a few more copy elements to update from a `grep`

```
Searching 390 files for "push open"

/Users/kevinowocki/Sites/gitcoin/web/app/dashboard/embed.py:
  205          ## config
  206          show_value, value = summarize_bounties(super_bounties)
  207:         text = '{}\nPush Open Source Forward | Powered by Gitcoin.co'.format(wrap_text(value, 45) if show_value else "")
  208          # execute
  209          draw = ImageDraw.Draw(img)

/Users/kevinowocki/Sites/gitcoin/web/app/dashboard/models.py:
  411            "type": "Organization",
  412            "email": "founders@gitcoin.co",
  413:           "bio": "Push Open Source Forward.",
  414            "gists_url": "https:\/\/api.github.com\/users\/gitcoinco\/gists{\/gist_id}",
  415            "company": null,

/Users/kevinowocki/Sites/gitcoin/web/app/retail/views.py:
   64      'title': 'Mission',
   65      'card_title': "Gitcoin is a mission-driven organization.",
   66:     'card_desc': "Our mission is to push open source forward.",
   67      'avatar_url': '/static/v2/images/mission/hero.png',
   68    }
   ..
  209  Gitcoin Core LLC is the legal entity that manages the software development of the Gitcoin Network (Gitcoin).
  210  
  211: The Gitcoin Network is a series of smart contracts that helps Push Open Source Forward, but enabling developers to easily post and manage funded issues.            """
  212          },
  213          {
  ...
  224              'q': 'What is the mission of Gitcoin Core?',
  225              'a': """
  226: The mission of Gitcoin is "Push Open Source Forward".
  227  
  228              """

/Users/kevinowocki/Sites/gitcoin/web/app/retail/templates/emails/bounty_roundup.html:
   78  
   79  <h3>Open Funded Issues</h3>
   80: <h5>Push Open Source Forward ⚡️ Make Extra Cash 💰</h5>
   81  
   82  <br>

/Users/kevinowocki/Sites/gitcoin/web/app/retail/templates/shared/cards_pic.html:
   23      <meta name="twitter:title" content="{{card_title}}" />
   24  {% else %}
   25:     <meta name="twitter:title" content="Gitcoin: Push Open Source Repos Forward" />
   26  {% endif %}
   27  {% if card_desc %}
   ..
   45      <meta property=og:title content="{{card_title}}">
   46  {% else %}
   47:     <meta property=og:title content="Gitcoin: Push Open Source Repos Forward.">
   48  {% endif %}
   49  <meta property=og:url content="https://gitcoin.co/" >
   ..
   55  {% endif %}
   56  <meta property=og:type content="website">
   57: <meta property=og:site_name content="Gitcoin: Push Open Source Repos Forward.">
   58  

/Users/kevinowocki/Sites/gitcoin/web/docs/CONTRIBUTING.md:
  128  
  129  <img src='https://d3vv6lp55qjaqc.cloudfront.net/items/263e3q1M2Y2r3L1X3c2y/helmet.png'/>
  130: Welcome to the gitcoin community. Lets push Open Source Forward.
  131  

10 matches across 6 files


```

**Comment by mbeacom** [comment]
This branch doesn't appear to be current with out Python 3 changes.

**Comment by cd10012** [comment]
Sorry I made another PR to address these issues since I'm having trouble saving this branch: https://github.com/gitcoinco/web/pull/551

**Pull request closed without merging**